src/raftstore/store/peer.rs

```diff
@@ -1208,12 +1128,16 @@ impl Peer {
 
     pub fn post_apply(
         &mut self,
-        groups: &mut HashSet<u64>,
         apply_state: RaftApplyState,
         applied_index_term: u64,
         merged: bool,
         apply_metrics: &ApplyMetrics,
-    ) {
+    ) -> bool {
```

Comment:
seems that the return value is used in nowhere

Reply:
The changes weren't complete, now it's in use.